src/tools/tests/golden_kt.arcs

```diff
@@ -40,4 +40,4 @@ particle KotlinPrimitivesGolden
       price: Float,
       stock: Int
     }>
-  } [(2n * big + 3n > 5n) and (num < 1000)]
+  } [(2n * big + 3n > 5n) and (5i*nt + 10i*nt > 30i) and (10l*lng >= 100000l and 10l*lng < 1000000l) and (num < 1000)]
```

Comment:
might be worth renaming 'nt' and 'lng'; they're distressingly close to 'i' and 'n' :)

Reply:
Done